go/libraries/doltcore/merge/merge_prolly_rows.go

```diff
@@ -99,22 +164,26 @@ func mergeTableData(ctx context.Context, tm TableMerger, finalSch schema.Schema,
 
 		switch diff.Op {
 		case tree.DiffOpDivergentModifyConflict, tree.DiffOpDivergentDeleteConflict:
+			// In this case, a modification or delete was made to one side, and a conflicting delete or modification
+			// was made to the other side, so these cannot be automatically resolved.
 			s.Conflicts++
-			err = conflicts.merge(ctx, diff)
-			if err != nil {
-				return nil, nil, err
-			}
-			err = sec.merge(ctx, diff)
+			err = conflicts.merge(ctx, diff, nil)
 			if err != nil {
 				return nil, nil, err
 			}
+			// TODO: Why would we merge the secondary index data if we can't merge the primary index?
+			//       This doesn't seem correct... was this needed for unique index validation or something?
+			//err = sec.merge(ctx, diff, nil)
+			//if err != nil {
+			//	return nil, nil, err
+			//}
```

Comment:
@max-hoffman – I wanted to ping you about this code before I pulled it out. It seems like we shouldn't be merging into the secondary indexes here if we don't merge data into the primary index, but I wanted to double check that I'm not just missing something.

Reply:
Yeah, secMerger doesn't seem to do anything with the diff. It looks like i was a little more careful with erroring in the default case for most of the handlers

Reply:
sweet; thanks for the sanity check! just wanted to make sure I wasn't misunderstanding some details about the 3way differ.